### DISCUSSION TITLE
你好请教一下 我想在docker上部署应该如何部署

### DISCUSSION THREAD

**Discussion by 972513547** [swuecho/chat#370] [Q&A] [answered]
你好请教一下 我是刚接触docker的 参照chatgpt-web-plus 没有搞明白如何部署 能否指明一下如何在docker上部署呢

**Comment by swuecho** [comment, marked as answer]
https://github.com/swuecho/chat/blob/master/docker-compose.yaml

把上面的文件放到一个文件夹, 然后改相应的 环境变量..
```
docker-compose up -d 
```

就可以了.

可能你需要学一下 docker-compse 的用法.
